core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -55,12 +54,6 @@
    *   <li>hadoop: org.apache.iceberg.hadoop.HadoopCatalog</li>
    * </ul>
    */
-  public static final String ICEBERG_CATALOG_TYPE = "type";
-  public static final String ICEBERG_CATALOG_TYPE_HADOOP = "hadoop";
-  public static final String ICEBERG_CATALOG_TYPE_HIVE = "hive";
-  public static final String ICEBERG_CATALOG_HIVE = "org.apache.iceberg.hive.HiveCatalog";
-  public static final String ICEBERG_CATALOG_HADOOP = "org.apache.iceberg.hadoop.HadoopCatalog";
```

Comment:
Instead of removing these now, can you deprecate them with a message about when they will be removed? That way we don't accidentally break anyone that references them.

Reply:
+1. People maintaining forks won't necessarily immediately cherry-pick this PR and it could then break their builds if they reference them.

Reply:
+1. People maintaining forks won't necessarily immediately cherry-pick this PR and it could then break their builds if they reference them.